test/show.jl

```diff
@@ -301,8 +301,8 @@ end
     @test sprint(show, df) == """
         1×3 DataFrame
          Row │ a           b                    c
-             │ Date        DateTime             Day
-        ─────┼────────────────────────────────────────
+             │ Date        DateTime             Dates.Day
```

Comment:
Just curious: why don't we print `Dates.DateTime`?

Anyway it would make sense to never print the module name to avoid this kind of variation (which is hard to understand for users).

Reply:
If that's doable, I'd say that taking into account the final size width of the column (including contents) would be better. Otherwise there's some wasted space which in some cases could be used to print relevant information.

Reply:
@ronisbr - is this EASILY doable? (if it is hard it is probably better to focus on HTML/LaTeX support first). Thank you!